ete4/coretype/tree.pyx

```diff
@@ -2126,47 +2126,14 @@ cdef class TreeNode(object):
         .. versionadded: 2.1
 
         Converts a tree into ultrametric topology (all leaves must have
-        the same distance to root). Note that, for visual inspection
-        of ultrametric trees, node.img_style["size"] should be set to
-        0.
```

Comment:
This comment is no longer needed for the new visualizer. Do we want to keep it? @jhcepas